framework/DataObjects/Data.py

```diff
@@ -639,7 +640,7 @@ def getParam(self,typeVar,keyword,nodeId=None,serialize=False):
     else:
       if typeVar.lower() in ['input','inputs','unstructuredInput']:
         returnDict = {}
-        if keyword in self._dataContainer['inputs'].keys() + self._dataContainer['unstructuredInputs'].keys():
+        if keyword in itertools.chain(self._dataContainer['inputs'].keys(),self._dataContainer['unstructuredInputs'].keys()):
```

Comment:
This makes sense as you don't have to construct the new object that mashes both lists together, also timing shows improvement:

```python
>>> import itertools, timeit
>>> timeit.timeit('[x for x in A+B]', setup = 'A = list(range(1000000)); B = list(range(1000000,2000000))', number=10)
0.8271918296813965
>>> timeit.timeit('[x for x in itertools.chain(A,B)]', setup = 'A = list(range(1000000)); B = list(range(1000000,2000000))', number=10)
0.6930079460144043
```